readme.md

```diff
@@ -89,23 +91,22 @@ var clown = {
 Access the value `"Sue"` out of the `names` array:
 
 ```js
-answer goes here
+names[2];
 ```
 
 ## Question 9
 
 Write a for loop such that you say hello(using `console.log`) to each name in the `names` array
 
 ```js
-answer goes here
+for(var i = 0; i < 4; i++)
+console.log("hello " + names[i]);
 ```
 
```

Comment:
while you did nail this, 4 is what is called a magic number in programming. if the array increased size, it wouldn't print anything past the 3rd index. the fix is to use `names.length` in place of the number `4`.

 you weren't expected to know that so this is just adding to the solid foundation you already have.